easybuild/easyconfigs/g/Geant4-data/Geant4-data-11.3.eb

```diff
@@ -25,6 +25,7 @@ local_datasets = [
     ('G4TENDL', '1.4', 'G4TENDL', 'G4PARTICLEHPDATA'),  # TENDL
     ('G4NUDEXLIB', '1.0', 'G4NUDEXLIB', 'G4NUDEXLIBDATA'),  # NuDEX
     ('G4URRPT', '1.1', 'G4URRPT', 'G4URRPTDATA'),  # G4URRPT
+    ('G4CHANNELING', '1.0', 'G4CHANNELING', 'G4CHANNELINGDATA'),  # G4CHANNELING
```

Comment:
Hrm. I dunno about this; we are mixing and matching things here and creating something that isn't version 11.3, not it is version 11.3.2 because that one has other updates as well. I would rather see Geant-data-11.3.2.eb with this included. 

As the comment in this easyconfig says:

> # Pick up the correct sets and versions from cmake/Modules/G4DatasetDefinitions.cmake

Reply:
oh nevermind, the link in the comment block was outdated, it wasn't pointing to 11.3, but to the older 11.2.2 still.